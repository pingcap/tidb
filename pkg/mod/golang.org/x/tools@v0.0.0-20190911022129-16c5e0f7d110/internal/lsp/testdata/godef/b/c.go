package b

// This is the in-editor version of the file.
// The on-disk version is in c.go.saved.

var _ = S1{ //@godef("S1", S1)
	F1: 99, //@godef("F1", S1F1)
}
