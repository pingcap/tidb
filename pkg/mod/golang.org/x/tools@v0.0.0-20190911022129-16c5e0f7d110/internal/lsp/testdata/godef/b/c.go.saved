package b

// This is the on-disk version of c.go, which represents
// the in-editor version of the file.

}

