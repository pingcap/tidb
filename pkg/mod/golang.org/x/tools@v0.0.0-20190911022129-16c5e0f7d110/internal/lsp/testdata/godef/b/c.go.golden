-- S1-definition --
godef/b/b.go:8:6-8: defined here as S1 struct {
	F1  int //@mark(S1F1, "F1")
	S2      //@godef("S2", S2), mark(S1S2, "S2")
	a.A     //@godef("A", A)
}

-- S1-definition-json --
{
	"span": {
		"uri": "file://godef/b/b.go",
		"start": {
			"line": 8,
			"column": 6,
			"offset": 196
		},
		"end": {
			"line": 8,
			"column": 8,
			"offset": 198
		}
	},
	"description": "S1 struct {\n\tF1  int //@mark(S1F1, \"F1\")\n\tS2      //@godef(\"S2\", S2), mark(S1S2, \"S2\")\n\ta.A     //@godef(\"A\", A)\n}"
}

-- S1-hover --
S1 struct {
	F1  int //@mark(S1F1, "F1")
	S2      //@godef("S2", S2), mark(S1S2, "S2")
	a.A     //@godef("A", A)
}
-- S1F1-definition --
godef/b/b.go:9:2-4: defined here as field F1 int

-- S1F1-definition-json --
{
	"span": {
		"uri": "file://godef/b/b.go",
		"start": {
			"line": 9,
			"column": 2,
			"offset": 215
		},
		"end": {
			"line": 9,
			"column": 4,
			"offset": 217
		}
	},
	"description": "field F1 int"
}

-- S1F1-hover --
@mark(S1F1, "F1")
field F1 int
